ote/src/cljs/ote/ui/list_header.cljs

```diff
@@ -1,12 +1,13 @@
 (ns ote.ui.list-header
   "Header component for OTE list pages"
   (:require [cljs-react-material-ui.reagent :as ui]
-            [ote.ui.common :as common]))
+            [ote.ui.common :as common]
+            [ote.style.base :as style-base]))
 
-(defn header [title add-button operator-selection user-guide-button]
+(defn header [app title add-button operator-selection user-guide-button]
   [:div
    (when title
-     [:div.row [:h1 title]])
+     [:div.row (when (< (:width app) style-base/mobile-width-px) {:style {:padding-top "20px"}}) [:h1 title]])
```

Comment:
Any chance of adding this clause into a media query? I'd avoid these kind of conditional "surprises" in code if possible and define a media query that does the job in a stylefy style definition.

Reply:
Didn't even think about using stylefy for this, but it worked quite nicely.